sysboundary/sysboundary.cpp

```diff
@@ -490,7 +490,10 @@ bool SysBoundary::classifyCells(dccrg::Dccrg<spatial_cell::SpatialCell, dccrg::C
             // Here for debugging since boundarytype should be fed from MPIGrid
             technicalGrid.get(x, y, z)->sysBoundaryFlag = sysboundarytype::N_SYSBOUNDARY_CONDITIONS;
             technicalGrid.get(x, y, z)->sysBoundaryLayer = 0;
-            technicalGrid.get(x, y, z)->maxFsDt = numeric_limits<Real>::max();
+            // Don't screw with maximum dt when refining
+            if (P::tstep == 0) {
```

Comment:
Does this need additional handling when restarting?

Reply:
`technicalGrid` is read on restart after boundary cell classification

Reply:
https://github.com/fmihpc/vlasiator/blob/dev/iowrite.cpp#L1659
No we only write and re-read fg_perturbed_b and fg_e.

Reply:
Changed to use `tstep_min`